bam_fastq.c

```diff
@@ -223,14 +237,11 @@ static bool parse_opts(int argc, char *argv[], bam2fq_opts_t** opts_out)
             case '2': opts->fnr[2] = optarg; break;
             case 'o': opts->fnr[1] = optarg; opts->fnr[2] = optarg; break;
             case 'f': opts->flag_on |= strtol(optarg, 0, 0); break;
-            case 'F':
-                if (!flag_off_set) {
-                    flag_off_set = 1;
-                    opts->flag_off = 0;
-                }
-                opts->flag_off |= strtol(optarg, 0, 0);
-                break;
```

Comment:
I believe this chunk is unnecessary and can be replaced by a simpler statement as shown below for better readability. `flag_off_set` has a default of zero so the `if` loop is always entered and `opts->flag_off` set to 0. This can be replaced by the line below.

This change is not essential to the objective of this PR.